daysinamonth.js

```diff
@@ -3,43 +3,72 @@
 // }
 
 export default class getDays {
-    constructor() {
-        this.yearDates = [];
-        this.yearIndexes = {};
-    }
+  constructor() {
+    this.yearDates = [];
+    this.yearIndexes = {};
+    this.yearWeekday = [];
+  }
 
-    getDaysInMonth(month, year) {
-        return new Date(year, month, 0).getDate();
-    }
+  getDaysInMonth(month, year) {
+    return new Date(year, month, 0).getDate();
+  }
 
-    fillDays(year) {
-        let index = 0;
-        for (let i = 1; i < 13; i++) {
-            let date = this.getDaysInMonth(i, year);
-            for (let j = 1; j <= date; j++) {
-                let month;
-                if (i < 10) {
-                    month = ['0'];
-                    month.push(i.toString())
-                    month = month.join("");
-                } else {
-                    month = ''+i+'';
-                }
-                let day;
-                if (j < 10) {
-                    day = ['0'];
-                    day.push(j.toString());
-                    day = day.join("");
-                } else {
-                    day = ''+j+'';
-                }
-                let dateString = `${year}-${month}-${day}`
-                this.yearDates.push(dateString);
-                this.yearIndexes[dateString] = index;
-                index++;
-            }
+  fillDays(year) {
+    let index = 0;
+    for (let i = 1; i < 13; i++) {
```

Comment:
Airbnb style guide calls for i + 1 or i += 1. Should change i++ to reflect that.
```suggestion
    for (let i = 1; i < 13; i += 1) {
```